optics/core/src/main/scala/tofu/optics/Equivalent.scala

```diff
@@ -3,29 +3,37 @@ package tofu.optics
 import cats._
 import cats.arrow._
 import cats.instances.function._
-import tofu.optics.data.{Constant, Tagged}
+import tofu.optics.data._
 
 /** aka Iso
   * S and B share same information
   */
-trait PEquivalent[-S, +T, +A, -B] extends PSubset[S, T, A, B] with PContains[S, T, A, B] {
+trait PEquivalent[-S, +T, +A, -B]
+    extends PSubset[S, T, A, B] with PContains[S, T, A, B] with PBase[PEquivalent, S, T, A, B]
```

Comment:
formatting became weird

Reply:
We will update to newer scalafmt, hopefully this would be better.
If not future rules solving #292 should solve it